packages/sanity/src/core/config/prepareConfig.ts

```diff
@@ -284,7 +284,12 @@ function resolveSource({
     // TODO: validate templates
     // TODO: validate that each one has a unique template ID
   } catch (e) {
-    errors.push(e)
+    throw new ConfigResolutionError({
+      name: config.name,
+      // TODO: figure out this name
```

Comment:
without any further context it's a bit unclear to me what the actionable thing is re this todo comment

Reply:
Copy+pasted from another location, I think we were figuring out the `space` vs `workspace` vs `source` thing. Will remove.